### DISCUSSION TITLE
Addon does not start, complains about missing module "contextvars"

### DISCUSSION THREAD

**Discussion by akuntzagk** [CastagnaIT/plugin.video.netflix#1379] [Q&A] [answered]
Hi,

I cannot start the plugin. Error is
`ModuleNotFoundError: No module named 'contextvars'
`
Kodi is 19.3, Plugin is v1.18.8+matrix, OS is Ubuntu 20.04 , python is 3.8.10
contextvars module is installed.
```
pip list |grep contextvars
contextvars             2.4             
```

Any idea why the python module path might be broken inside kodi?

**Comment by CastagnaIT** [comment, marked as answer]
sorry for the delay
its possible that you miss to install pycryptodomex

`pip install pycryptodomex`
